perfrunbook/debug_system_perf.md

```diff
@@ -125,3 +125,59 @@ When running Java applications, monitor for differences in behavior using JFR (J
   # Capture output from x86 and Graviton2 and then diff the files
   ```
 
+## Check Memory behavior via `sar -r`
+
+If your benchmark involves reading and writing to file, for example database workloads, check for "dirty pages" and "page cache" via `kbdirty`, `kbcached` respectively between two systems. This gives important information if your system is getting limited by page cache or not compared to other.
+
+**Dirty Pages**: Is the data stored in page cached waiting to be written to the underlying storage device.
```

Comment:
Show example of the tool output and then the fields that should be checked and how to interpret them as discrete steps.

Make sure it is in checklist format, 
1. Do X
2. Measure Y
3. Compare A and B
...